corehq/apps/saved_reports/tests/test_scheduled_reports.py

```diff
@@ -204,7 +204,7 @@ def test_get_scheduled_report_response(self):
             hour=12, minute=None, day=30, interval='monthly', config_ids=[report_config._id]
         )
         report.save()
-        response = get_scheduled_report_response(
+        report_text = get_scheduled_report_response(
             couch_user=self.user, domain=domain, scheduled_report_id=report._id
         )[0]
-        self.assertTrue(self.user.username in response.decode('utf-8'))
+        self.assertTrue(self.user.username in report_text)
```

Comment:
Just naively grepping, there are other places where we `decode('utf-8')` in tests on a `response` variable, mostly API calls (HQ APIs, SMS, case claim). Do you think those could point to similar issues that deserve followup? (Not in this PR.)

Reply:
Possibly. We may want to look into them.